jets-bench/src/data_structures.rs

```diff
@@ -185,12 +185,13 @@ impl SimplicityEncode for elements::confidential::Asset {
     fn value(&self) -> Arc<Value> {
         match self {
             elements::confidential::Asset::Explicit(a) => {
-                Value::right(Value::u256(&a.into_inner().0))
+                Value::right(Value::u256(a.into_inner().to_byte_array()))
             }
             elements::confidential::Asset::Confidential(gen) => {
                 let ser = gen.serialize();
                 let odd_gen = ser[0] & 1 == 1;
-                let x_bytes: &[u8; 32] = (&ser[1..33]).try_into().unwrap();
+                let mut x_bytes = [0u8; 32];
+                x_bytes.copy_from_slice(&ser[1..33]);
```

Comment:
In 753d70a38eb65935bdb798872eb9cf6400f4a4d5:

You can definitely keep the old `try_into().unwrap()` logic. All you need to do is change `&[u8; 32]` to `[u8; 32]`.

Repeated many times.

Reply:
Addressed